### ISSUE TITLE
Cannot parse graphite metrics when there is an empty space at the end of the metrics

### ISSUE THREAD

**Issue by cjagus** [VictoriaMetrics/VictoriaMetrics#1865]
**Describe the bug** 
We have few internal applications directly sending data to graphite (not via statsd), noticed victoria-metrics Cannot parse graphite metrics when there is an empty space at the end of the metrics, this seems to work before in graphite-cache and go-carbon servers. 

**To Reproduce**
```
echo "foo.bar.baz 123 `date +%s` " | nc -N localhost 2003
echo "foo.bar.baz;tag1=value1;tag2=value2 123 `date +%s` " | nc -N localhost 2003
```
**Expected behavior**
Can we Trim whitespace line from end of the string, tried in my local fork and it seems working fine,  

**Logs**
vm[7591]: 2021-11-28T10:52:33.319Z	error	VictoriaMetrics/lib/protoparser/graphite/parser.go:158	cannot unmarshal Graphite line "host.aerospike.customers.temp.migrate_tx_partitions_initial 37 1638096753 ": cannot unmarshal timestamp from "1638096753 ": cannot parse float64 from "1638096753 "

**Version**
```
$ ./victoria-metrics-prod --version
victoria-metrics-20211108-134932-tags-v1.69.0-0-g69e655ba7
```

**Used command-line flags**
```
/opt/victoria-metrics/bin/victoria-metrics-prod -storageDataPath /opt/victoria-metrics/data -graphiteListenAddr=:2003 -selfScrapeInterval=30s
```

**Comment by valyala** [comment]
The issue should be addressed in the commit 49a18b86605c5d92da1ab9d5d46713f482aa6945. This commit will be included in the next release of VictoriaMetrics.

**Comment by valyala** [comment]
The commit 49a18b8 has been included in [v1.70.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the feature request as done.

**Comment by cjagus** [comment]
Thanks @valyala, works fine in v1.70.0.
